setup.py

```diff
@@ -183,14 +250,17 @@ def run(self):
     packages=find_packages(
         exclude=["docs", "docs-src", "integration_tests", "tests", "tests.*", "tutorial"]
     ),
-    install_requires=["aiohttp>3.5.2,<4.0.0"],
+    install_requires=[
+        "aiohttp>3.5.2,<4.0.0"  # TODO: move to extras_require in v3
```

Comment:
As I mentioned in the issue, I'm going to remove aiohttp from the required dependencies in v3.